src/main/java/Rica.java

```diff
@@ -0,0 +1,154 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Rica {
```

Comment:
Rica class is doing multiple things such as adding marking and printing tasks and separating into different classes can be used for better clarity